core/storage/learner_group/gae_models.py

```diff
@@ -205,11 +205,7 @@ def export_data(cls, user_id: str) -> Dict[str, LearnerGroupDataDict]:
                     'subtopic_page_ids': [],
                     'story_ids': []
                 }
-            # We add a no branch flag here because if found_models is not None,
-            # user_id has to be either in learner_user_ids,
-            # invited_learner_user_ids or facilitator_user_ids.
-            elif user_id in ( # pragma: no branch
-                    learner_group_model.facilitator_user_ids):
+            else:
```

Comment:
Below these "else" statements, add a comment like "# To get to this branch, the user_id would need to be in facilitator_user_ids."

You can also add an assertion of that so that any errors at least get surfaced somehow.

Reply:
Done